static/js/tools/timeline_util.ts

```diff
@@ -170,6 +176,28 @@ function getStatsVarInfo(dcids) {
   });
 }
 
+function getStatsVar(dcids: string[]) {
+  if (dcids.length === 0) {
+    return Promise.resolve([]);
+  }
+  const promises = [];
+  for (const dcid of dcids) {
+    promises.push(
+      axios.get("/api/place/statsvars/" + dcid).then((resp) => {
+        return resp.data;
+      })
+    );
+  }
+  return Promise.all(promises).then((values) => {
+    let statvars = ["Count_Person"]; // Count_Person not in List ???
```

Comment:
can use javascript set here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set, should be faster with large number items

Reply:
done.